portafly/doc/ux_specs/backend/create_backend.md

```diff
@@ -0,0 +1,44 @@
+# Create Backend
+
+[Back to index](../index.md)
+
+* [Mockups](https://marvelapp.com/prototype/12844cg4)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/APPDUX-343)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-5575)
+
+##### Title
+* Create a new backend
```

Comment:
According to the IBM style guide, `backend` should either be written as `back end` (noun) or `back-end` (adjective), but if there is a more specific term like "server" for example, then it should be replaced.

Reply:
@dfennessy This is a good point, thank you! We're actually discussing about removing the term altogether. I would leave it as it is for now just for consistency with the current term. It will be edited as soon as we make a decision.